src/aggregator/integration/setup.go

```diff
@@ -263,54 +272,18 @@ func newTestServerSetup(t *testing.T, opts testServerOptions) *testServerSetup {
 }
 
 func (ts *testServerSetup) newClient(t *testing.T) *client {
+	iOpts := ts.opts.InstrumentOptions()
+	iOpts = iOpts.SetLogger(iOpts.Logger().With(zap.Bool("client", true)))
```

Comment:
Is there any purpose for this `zap.Bool("client", true)`? Do we have it as `false` anywhere?